cronexpr_parse.go

```diff
@@ -210,6 +214,55 @@ var cronNormalizer = strings.NewReplacer(
 	"@daily", "0 0 0 * * * *",
 	"@hourly", "0 0 * * * * *")
 
+var systemdNormalizer = strings.NewReplacer(
+	"minutely", "*-*-* *:*:00",
+	"hourly", "*-*-* *:00:00",
+	"daily", "*-*-* 00:00:00",
+	"monthly", "*-*-01 00:00:00",
+	"weekly", "Mon *-*-* 00:00:00",
+	"yearly", "*-01-01 00:00:00",
+	"annually", "*-01-01 00:00:00",
+	"quarterly", "*-01,04,07,10-01 00:00:00",
+	"semiannually", "*-01,07-01 00:00:00",
+)
+
+type FieldType uint8
+
+const (
+	WeekDayField FieldType = 0
+	DayField     FieldType = 1
+	TimeField    FieldType = 2
+)
+
+var systemdFieldsSig = map[FieldType]string{
+	WeekDayField: `(mon|tue|wed|thu|fri|sat|sun){1,3}`,
+	DayField:     `([\d,\*\./]{0,4}-)+([\d,\*\./]{0,2}){0,2}`,
+	TimeField:    `([\d\*\.,/]+:){1,2}[\d\*\.,/]+`,
+}
+
+/******************************************************************************/
+
+func (expr *Expression) normalyzeSystemd() error {
+	// Maybe one of the built-in aliases is being used
+	expr.expression = systemdNormalizer.Replace(expr.expression)
+	expr.expression = strings.ToLower(expr.expression)
```

Comment:
Do systemd have restriction on space count?
Is record like this is valid? `*-01-01         00:00:00`
If yes we should remove duplicate spaces.